ChatQnA/tests/test_chatqna_pinecone_on_xeon.sh

```diff
@@ -0,0 +1,258 @@
+#!/bin/bash
```

Comment:
In the new test structure, the test scripts name should be `ChatQnA/tests/test_compose_pinecone_on_xeon.sh`